charts/kubeit-app-of-apps/templates/applications.yaml

```diff
@@ -65,7 +65,7 @@ apiVersion: argoproj.io/v1alpha1
 kind: Application
 metadata:
   name: {{ $appTruncName }}
-  namespace: argocd
+  namespace: {{ $.Values.managementNamespace }}
```

Comment:
I cannot find managementNamespace in values.yaml. Is it expected?
And maybe chart version increment is required.

Reply:
check #63 line - https://dev.azure.com/dnvgl-gssit/Kubernetes%20Platform/_git/voa-platform-apps/pullrequest/23096?_a=files&path=/argocd/bootstrap-tenants-apps/templates/argo-applications.yaml

Reply:
I mean it is not mentioned here: https://github.com/dnv-gssit/kubeit-charts/blob/feature/argocd_2_5/charts/kubeit-app-of-apps/values.yaml
However it is only comment for advice.